### PR TITLE
Refs #21424 - Pin factory_girl_rails to ~> 4.8.0

### PR DESCRIPTION
**Pull request by mmoll**
Version 4.9.0 emits a deprecation warning which breaks the CI tests.
Rather than upgrade the stable branch it's easier and less disruptive to
pin.

### CODE REVIEW

**Comment by mmoll** [comment]
This is for 1.15-stable. There are some plugin tests (e.g. for discovery 9.1) that run against 1.15.

**Comment by mmoll** [comment]
Added another commit to fix the JS test on travis.

**Comment by tbrisker** [comment]
@mmoll I assume that the katello failure is to be expected?

**Comment by mmoll** [comment]
@tbrisker yes, because the current Katello master is tested there against Foreman 1.15

**Review by tbrisker — APPROVED** [review]
ACK from me, but I'll leave it to @dLobatog who is the release manager for 1.15 branch to pull the trigger.

**Review by dLobatog — APPROVED** [review]
Thanks @mmoll @tbrisker

**Pull request merged**
